### ISSUE TITLE
Opening shared files via Nextcloud-App only in pop-up

### ISSUE THREAD

**Issue by Bingenberger** [nextcloud/talk-ios#531]
For this issue, I compare the Android app with the iOS app:

In the Android app, when I tap on an odt file (or other editable format) that was shared with me via Talk, the file opens directly in the connected Collabora and I can edit it just fine.

In the iOS app, I have to select it directly with a long tap, then choose "Open in nextcloud" from the context menu. That also works, but the file is opened there in a popup. I can't edit the file from this screen pop, because Collabora also opens in the screen pop - and therefore doesn't use the full width of the screen.

I would prefer if the file is displayed directly in the Nextcloud app - without popup - or alternatively, the connected Collabora can be opened directly from the Talkapp, just like in the Android app.

**Comment by nickvergessen** [comment]
So currently Android is working on copying the iOS app, but maybe on types not mentioned in https://github.com/nextcloud/talk-android/issues/1098#issuecomment-822234676 it makes sense to go the other way? Not sure

**Comment by SystemKeeper** [comment]
I don't think this is a problem of the talk-ios app. When using "Open in nextcloud" we just switch to the iOS app and the file is opened there. So handling of different types (eg. fullscreen collabora) should be done there. 
Also there's already a report about this https://github.com/nextcloud/ios/issues/1558
